gst/nnstreamer/tensor_filter/tensor_filter_common.h

```diff
@@ -103,6 +103,12 @@ typedef struct _GstTensorFilterPrivate
   gboolean is_updatable; /**<  a given model to the filter is updatable if TRUE */
   GstTensorsConfig in_config; /**< input tensor info */
   GstTensorsConfig out_config; /**< output tensor info */
+  GList *in_combi; /**< Select the input tensor(s) to invoke the models */
+  GList *out_combi_i; /**< Select the output tensor(s) from the input tensor(s) */
+  GList *out_combi_o; /**< Select the output tensor(s) from the model output */
+  gboolean in_combi_defined; /**< True if input combination is defined */
+  gboolean out_combi_i_defined;/**< True if output combination from input is defined */
+  gboolean out_combi_o_defined;/**< True if output combination from model output is defined */
```

Comment:
For readability, how about defining extra data structure for those variables? (e.g., `GstTensorConbination`)

Reply:
Thank you for the suggestion. I agreed with your opinion. Updated.